### ISSUE TITLE
Multiscrape not working after most recent update

### ISSUE THREAD

**Issue by duffyd** [danieldotnl/ha-multiscrape#116]
I just upgraded to the latest HACs version of multiscrape and my sensors now no longer work.

## Version of the custom_component

v6.2.1

## Configuration

```yaml
multiscrape:
  - resource: https://covidlive.com.au/report/daily-cases/nsw
    scan_interval: 3600
    headers: 
      User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_2) AppleWebKit/601.3.9 (KHTML, like Gecko) Version/9.0.2 Safari/601.3.9
    sensor:
      - unique_id: nsw_covid_total 
        name: NSW COVID Cases - Total
        select: ".DAILY-CASES tr:nth-child(2) .COL3.CASES"
        icon: mdi:virus
        #value_template: "{{ value and value or '' }}"

      - unique_id: nsw_covid_today
        name: NSW COVID Cases - Today
        select: ".DAILY-CASES tr:nth-child(2) .COL5.NET"
        icon: mdi:virus
        #value_template: "{{ value and value or '' }}"

      - unique_id: nsw_covid_last_update
        name: NSW COVID Cases - Last Update
        select: ".DAILY-CASES tr:nth-child(2) .COL1.DATE"
        icon: mdi:virus
        #value_template: "{{ value and value or '' }}"
```

## Describe the bug

I've been running multiscrape for some time and the above sensors have been working fine. After upgrading to v6.2.1, multiscrape is no longer functioning and errors as per the error below.

## Debug log

```text

This error originated from a custom integration.

Logger: homeassistant.loader
Source: custom_components/multiscrape/sensor.py:40 
Integration: Multiscrape scraping component (documentation, issues) 
First occurred: 8:15:46 PM (3 occurrences) 
Last logged: 8:15:46 PM

Unexpected exception importing platform custom_components.multiscrape.sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/loader.py", line 603, in get_platform
    cache[full_name] = self._import_platform(platform_name)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 620, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
  File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/config/custom_components/multiscrape/sensor.py", line 40, in <module>
    discovery_info: DiscoveryInfoType | None = None,
TypeError: unsupported operand type(s) for |: 'types.GenericAlias' and 'NoneType'

```

**Comment by fl380** [comment]
same error here

**Comment by Herbs851** [comment]
Hi all,

Same error here.

**Comment by gvssr** [comment]
Same here!

**Comment by fl380** [comment]
v6.2.1 seems to now have been removed from HACS. 

To revert to v6.2.0, I removed Multiscrape integration via HACS and then re-installed.

**Comment by danieldotnl** [comment]
Yeah, sorry for that. Don't know what's happening yet, so I pulled back that release.

**Comment by Sjorsa** [comment]
I've been getting the error: `Invalid config for [multiscrape]: [index] is an invalid option for [multiscrape]. Check: multiscrape->multiscrape->0->sensor->0->index.` lately. Did something change about the configuration in a recent update? Is this related to this issue?

**Comment by danieldotnl** [comment]
Fixed in v6.2.2

@Sjorsa This is not related. Index feature has been removed since 6.0.0. (this might help you: https://github.com/danieldotnl/ha-multiscrape/issues/111)

**Comment by duffyd** [comment]
I can confirm my multiscrape sensors are online again. Many thanks!
